ngraph/core/src/os/lin/lin_so_loader.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <dlfcn.h>
+
+#include <iostream>
+#include <memory>
+
+#include "ngraph/check.hpp"
+#include "ngraph/file_util.hpp"
+#include "so_loader.hpp"
+
+namespace ov {
+
+class SOLoader::Impl {
```

Comment:
what about https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/plugin_api/shared_object.hpp

Reply:
Seems we need move it to ov directly
```c++
ov::core|runtime::SO -> ov::so
```